src/js/components/Alert.js

```diff
@@ -3,9 +3,10 @@ import React from 'react';
 
 import Icon from './Icon';
 
-const Alert = ({children, showIcon, type}) => {
+const Alert = ({children, flushBottom, showIcon, type}) => {
   const classes = classNames('alert', {
-    [`alert-${type}`]: type != null
+    [`alert-${type}`]: type != null,
+    'flush-bottom': flushBottom === true
```

Comment:
Both checks look redundant. 
Not empty string in `type` will render `alert-<type>`
Truthy value in `fulshBottom` will render `flush-bottom`